callouts/ssl_creds/localhost.crt

```diff
@@ -1,19 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
why are these files being deleted?

Reply:
the new structure is using the creds from python/extproc/ssl_creds, that's why we removed the external one

Reply:
That additional ssl_creds was to be used in other callout examples, but I can add them later, I am working on a proto generation script currently.